src/platform/amd/renoir/include/platform/chip_registers.h

```diff
@@ -0,0 +1,571 @@
+	unsigned int	:6;
+	} bits;
+unsigned int	u32all;
+} acp_bt_tx_intr_watermark_size_t;
+
+typedef	union acp_i2stdm_ier {
+	struct {
+	unsigned int	i2stdm_ien:1;
+	unsigned int	:31;
+	} bits;
+unsigned int	u32all;
+} acp_i2stdm_ier_t;
+typedef	union acp_i2stdm_irer {
+	struct {
+	unsigned int	i2stdm_rx_en:1;
+	unsigned int	i2stdm_rx_protocol_mode:1;
+	unsigned int	i2stdm_rx_data_path_mode:1;
+	unsigned int	i2stdm_rx_samplen:3;
+	unsigned int	i2stdm_rx_status:1;
+	unsigned int	:25;
+
+	} bits;
+unsigned int	u32all;
+} acp_i2stdm_irer_t;
+
+typedef	union acp_i2stdm_iter {
+	struct {
+	unsigned int	i2stdm_txen:1;
+	unsigned int	 i2stdm_tx_protocol_mode:1;
+	unsigned int	i2stdm_tx_data_path_mode:1;
+	unsigned int	i2stdm_tx_samp_len:3;
+	unsigned int	i2stdm_tx_status:1;
+	unsigned int	:25;
+	} bits;
+unsigned int	u32all;
+} acp_i2stdm_iter_t;
+
+typedef	union acp_bttdm_ier {
+	struct {
+	unsigned int	bttdm_ien:1;
+	unsigned int	:31;
+	} bits;
+unsigned int	u32all;
+} acp_bttdm_ier_t;
+
+typedef	union acp_bttdm_irer {
+	struct {
+	unsigned int	bttdm_rx_en:1;
+	unsigned int	bttdm_rx_protocol_mode:1;
+	unsigned int	bttdm_rx_data_path_mode:1;
+	unsigned int	bttdm_rx_samplen:3;
+	unsigned int	bttdm_rx_status:1;
+	unsigned int	:25;
+	} bits;
+unsigned int	u32all;
+} acp_bttdm_irer_t;
+
+typedef	union acp_bttdm_iter {
+	struct {
+	unsigned int	bttdm_txen:1;
+	unsigned int	bttdm_tx_protocol_mode:1;
+	unsigned int	bttdm_tx_data_path_mode:1;
+	unsigned int	bttdm_tx_samp_len:3;
+	unsigned int	bttdm_tx_status:1;
+	unsigned int	:25;
+	} bits;
+unsigned int	u32all;
+} acp_bttdm_iter_t;
+
+typedef	union acp_wov_pdm_dma_enable {
+	struct {
+	unsigned int	pdm_dma_en:1;
+	unsigned int	pdm_dma_en_status:1;
+	unsigned int	:30;
+	} bits;
+unsigned int	u32all;
+} acp_wov_pdm_dma_enable_t;
+
+typedef	union acp_wov_rx_ringbufaddr {
+	struct {
+	unsigned int	rx_ringbufaddr:27;
+	unsigned int	:5;
+	} bits;
+unsigned int	u32all;
+} acp_wov_rx_ringbufaddr_t;
+
+typedef	union acp_wov_rx_ringbufsize {
+	struct {
+	unsigned int	rx_ringbufsize:26;
+	unsigned int	:6;
+	} bits;
+unsigned int	u32all;
+} acp_wov_rx_ringbufsize_t;
+
+typedef	union acp_wov_rx_intr_watermark_size {
+	struct {
+	unsigned int	rx_intr_watermark_size:26;
+	unsigned int	:6;
+	} bits;
+unsigned int	u32all;
+} acp_wov_rx_intr_watermark_size_t;
+
+typedef	union acp_wov_pdm_no_of_channels {
+	struct {
+	unsigned int	pdm_no_of_channels:2;
+	unsigned int	:30;
+	} bits;
+unsigned int	u32all;
+} acp_wov_pdm_no_of_channels_t;
+
+typedef	union acp_wov_pdm_decimation_factor {
+	struct {
+	unsigned int	pdm_decimation_factor:2;
+	unsigned int	:30;
+	} bits;
+unsigned int	u32all;
+} acp_wov_pdm_decimation_factor_t;
+typedef	union acp_wov_misc_ctrl {
+	struct {
+	unsigned int	:3;
+	unsigned int	pcm_data_shift_ctrl:2;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:1;
+	unsigned int	:17;
+	unsigned int	:1;
+	unsigned int	:1;
```

Comment:
This file looks copy and pasted from somewhere else, but it look like it can be cleaned up to improve readability. i.e. the bitfield could be ```unsigned int :26;``` instead of above as these look like test or reserved bits that should not be touched by users.

Reply:
Addressed the Review Comment. Done